org.eclipse.scanning.device.ui/src/org/eclipse/scanning/device/ui/model/TypeEditor.java

```diff
@@ -143,7 +170,7 @@ private Bundle getOSGiBundle(String symbolicName, BundleContext bcontext) {
 		return null;
 	}
 
-	public void deactivate() throws Exception {
+	private void deactivate() throws Exception {

```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Define and throw a dedicated exception instead of using a generic one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00112)